packages/iota/src/utils/validationHelper.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+import type { AddressTypes } from "../models/addresses/addressTypes";
+import type { IRent } from "../models/IRent";
+import { BASIC_OUTPUT_TYPE, IBasicOutput } from "../models/outputs/IBasicOutput";
+import { ADDRESS_UNLOCK_CONDITION_TYPE } from "../models/unlockConditions/IAddressUnlockCondition";
+import type { IValidationResult } from "../validation/result";
+import { TransactionHelper } from "./transactionHelper";
+
+/**
+ * Class to help with validation.
+ */
+export class ValidationHelper {
+
+    /**
+     * Validate that the values in Set are unique.
```

Comment:
The values in a Set are unique by definition. Suggesting `Validates the array is composed of distinct elements'